README.md

```diff
@@ -272,6 +272,12 @@ Sign3Intelligence.getInstance(this).getIntelligence(object : IntelligenceListene
 
 
 ## Changelog
+### 3.1.1
+ - Fixed crash issue in android 6 & 7 devices.
+ - Fixed native code issue on some devices.
+ - Fixed camera supported FPS, BioMetricStatus issue.
+ - Now, EventMetrics will be triggered only in case of failure.
+ - Sign3 APIs now require a minimum SDK version of 23; apps targeting below this version must enclose Sign3 API calls within conditional checks.
```

Comment:
if we have added this, shouldn't this be a minor version bump instead of patch bump? 3.2 instead of 3.1.1

Reply:
Yes, that would be better.